applications/StructuralMechanicsApplication/custom_python/structural_mechanics_python_application.cpp

```diff
@@ -279,8 +279,20 @@ PYBIND11_MODULE(KratosStructuralMechanicsApplication,m)
     KRATOS_REGISTER_IN_PYTHON_3D_VARIABLE_WITH_COMPONENTS(m, EULER_ANGLES)
     KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, LAYER_EULER_ANGLES)
     KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, FATIGUE_REDUCTION_FACTOR)
-    KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, NUMBER_OF_CYCLES)
-    KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, WOHLER_STRESS)
+    KRATOS_REGISTER_IN_PYTHON_VARIABLE(m, NUMBER_OF_CYCLES) 
```

Comment:
see #8392 can you please move the variables to the ConstitutiveLawsApp?

Reply:
Done! I have only moved the variables affecting the fatigue CL; I will work on another PR with @AlejandroCornejo to move the other variables to the CL application.